webapp/platform/client/src/client4.ts

```diff
@@ -2345,13 +2345,14 @@ export default class Client4 {
 
     // General Routes
 
-    ping = () => {
+    ping = (getServerStatus: boolean, deviceId?: string) => {
         return this.doFetch<{
             status: string;
             ActiveSearchBackend: string;
+            FileStoreSatus: string;
         }>(
-            `${this.getBaseRoute()}/system/ping?time=${Date.now()}`,
```

Comment:
`time` is an unknown field for the server, hence it gets removed.

Reply:
Is there any caching on the ping API that this was being used to bust? That's what it looks like it was doing before

Reply:
I can't find any caching on the server endpoint :shrug: I don't know why the  `time` field was added.

Reply:
I guess that's probably a legacy thing. We used to be a lot more aggressive with adding `Cache-Control` headers to things